src/sysext/sysext.c

```diff
@@ -784,7 +894,20 @@ static int merge(Hashmap *images) {
         if (r < 0)
                 return r;
 
-        return r != 123; /* exit code 123 means: didn't do anything */
+        if (r == 123) /* exit code 123 means: didn't do anything */
+                return 0;
+
+        r = need_reload();
+        if (r < 0)
+                return r;
+
```

Comment:
please drop this empty line.

we generally try to keep function calls and their immediate error/return code handling together (and place empty lines before function calls, and after handling all their errors instead). hence please drop this empty line, since you continue handling the return code below.